src/charm.py

```diff
@@ -330,7 +329,7 @@ def _create_discourse_environment_settings(self) -> typing.Dict[str, str]:
             # I need to take the required envVars for the application to work properly
             "CONTAINER_APP_NAME": "discourse",
             "CONTAINER_APP_ROOT": "/srv/discourse",
-            "CONTAINER_APP_USERNAME": "discourse",
+            "CONTAINER_APP_USERNAME": "root",
```

Comment:
Is this due to rocks defaulting to root? There is the possibility to add a non-root user now as [annonunced](https://discourse.ubuntu.com/t/unifying-user-identity-across-snaps-and-rocks/36469/2) and described [here](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/reference/rockcraft.yaml/#run-user). It would be “_daemon_” (with UID/GID 584792), so might take some adjustments, though better than root in the long run. This can be done in a different PR, we just need to keep in the backlog.

Reply:
Second this, others look good to me, thanks!